### DISCUSSION TITLE
a "add to cart" like system

### DISCUSSION THREAD

**Discussion by NEMOxd2010** [modrinth/.github#113] [Feature requests]
make it so every mod (or any project) have a button that says smth like "add to list" and when u press it the mod will just go to a list and u can open this list using a button and what this list allows u to do is basically:
1. download all mods: this button will just download all the mods in the list in a zip file
2. follow all mods: follows every mod in the list
3. delete all mods: if u wanna reset the list just press this and eveything will be deleted
![Screenshot 2023-06-08 170919](https://github.com/modrinth/.github/assets/115355838/e176ad7a-265a-4c89-a291-1c4dc1e950ac)

**Comment by triphora** [comment]
Duplicate of #30

**Reply by NEMOxd2010** [reply]
yeah sorry i didnt see that suggestion
